modelci/types/do/model_do.py

```diff
@@ -46,6 +46,8 @@ class ModelDO(Document):
     weight = FileField()
     # Model task
     task = IntField(required=True)
+    # Parent Model ID
+    parent_model_id = StringField(required=True)
```

Comment:
This field is nullable. The 'root' model does not have a parent.

Reply:
The default value of  'root' model's parent_model_id will be an empty string, and we could extract root models by querying an empty parent_model_id

Reply:
> The default value of 'root' model's parent_model_id will be an empty string, and we could extract root models by querying an empty parent_model_id

Setting this field as nullable is more convenient, I will fix that.